### PR TITLE
tbg: Fix existing-folder error message to stderr

### PR DESCRIPTION
**Pull request by jkelling**
EDITED by @PrometheusPi:
Stream error message to stderr and fix minor typos.

### CODE REVIEW

File: src/tools/bin/tbg [hunk 1]
```diff
@@ -430,7 +430,7 @@ if [ -d "$job_outDir" ] ; then
     if [ "$force" == "true" ] ; then
         echo "Warning: using existing folder on user-request [-f]"
     else
-        echo "destination path already in use, can't create new folder"
+        echo "Destination path already in use, can't create new folder" >&2
```
Comment (PrometheusPi) [inline src/tools/bin/tbg]: OPTIONAL: 
If you want a capital letter to start a sentence please ad a dot in the end too. 
Good point with streaming this to stderr.

**Comment by jkelling** [comment]
Regarding capitalization: I wanted to be consistent withe other messages in tbg wich start with capital letter and do not end with a period. They would all need to be changed.

File: src/tools/bin/tbg (hunk 1 above)
Comment (PrometheusPi) [reply src/tools/bin/tbg]: regarding @jkelling reply below: I can accept that - I will merge as soon as the compile test is possitive

Comment (ax3l) [reply src/tools/bin/tbg]: please write `cannot` while we are on it.

**Pull request merged**